libqtile/confreader.py

```diff
@@ -99,6 +99,25 @@ def update(self, *, fake_screens=None, **settings):
                 value = getattr(self, key, default[key])
             setattr(self, key, value)
 
+    def reload_config_submodules(self, path):
+        """Reloads python files from same folder as config file."""
+        to_reload = []
+        folder = path.parent
+        for name, module in sys.modules.items():
+
+            # Skip built-ins and anything with no filepath.
+            if getattr(module, "__file__", False):
```

Comment:
```suggestion
            if hasattr(module, "__file__"):
```

Reply:
Wanted to use walrus operator here but realised I couldn't because of pypy3.7 but forgot to change this to `hasattr`.

Reply:
Ah, gotcha.